packages/core/i18n/nls.json

```diff
@@ -179,6 +179,8 @@
       }
     },
     "filesystem": {
+      "prepareDownloadLink": "Preparing download link...",
+      "prepareDownload": "Preparing download...",
```

Comment:
Minor: Adding the default value to the `nls.json` is not necessary, as the automated translation update (see https://github.com/eclipse-theia/theia/pull/12011) will pick up the value automatically.

Reply:
fixed